packages/uma-auth-client/src/hooks/useOAuth.tsx

```diff
@@ -310,7 +335,7 @@ const processAsUmaAuthToken = (
   };
 
   if (token.nwc_expires_at && typeof token.nwc_expires_at === "number") {
-    umaToken.nwc_expires_at = token.nwc_expires_at;
+    umaToken.nwc_expires_at = token.nwc_expires_at * 1000;
```

Comment:
I noticed we're not consistent on seconds vs ms in this regard. We probably should be, right? Likely a backend thing